webpack.config.js

```diff
@@ -124,21 +126,22 @@ if (ENV === 'development') {
         {
           test: /\.jsx?$/,
           exclude: /(node_modules|bower_components)/,
-          use: [
-            {
-              loader: 'babel-loader',
-              options: {
-                presets: ['@babel/preset-env'],
-              },
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: ['@babel/preset-env'],
             },
-          ],
+          },
         },
         {
           test: /\.scss?$/,
-          use: [
-            'style-loader',
-            'css-loader',
-            `sass-loader?includePaths=${sassPaths}`,
+          use: ['style-loader',
+            { loader: 'css-loader' },
```

Comment:
You can probably use just `'css-loader'` here, right? Not a big deal.

Reply:
Not sure why, but the styles do not work without `style-loader`

Reply:
Oh, I meant `'css-loader'` instead of `{ loader: 'css-loader' }`.

Reply:
Ah yes. That makes sense.